src/main/java/tds/content/controllers/ContentController.java

```diff
@@ -51,11 +55,11 @@ public ContentController(final ContentService contentService) {
     @ResponseBody
     public ResponseEntity<ITSDocument> getItemDocument(@RequestParam final String itemPath, @RequestParam(required = false) final String contextPath,
                                                        @RequestParam(required = false) final boolean oggAudioSupport,
-                                                       @RequestBody final AccLookup accLookup) {
+                                                       @RequestBody final AccLookup accLookup) throws IOException {
```

Comment:
What is the behavior here if we throw the IOException?  Should we be catching it and instead throwing an IllegalStateException?  Or an illegal argument exception?  It seems like we should throw an appropriate http error code rather than a 500.

Also, I think it is time that ContentService leverages the common Exception advice.  There is enough exception handling and throwing in this application that it seems like it is warranted.